config/services_prod.yaml

```
@@ -0,0 +1,4 @@
+parameters:
+
+  .container.dumper.inline_factories: true
+  debug.container.dump: false
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)